packages/miniflare/src/workers/assets/rpc-proxy.worker.ts

```diff
@@ -0,0 +1,33 @@
+import { WorkerEntrypoint } from "cloudflare:workers";
+
+export default class SpecialDefault extends WorkerEntrypoint {
+	fetch(request) {
+		return this.env.ROUTER_WORKER.fetch(request);
+	}
+
+	constructor(...args) {
+		return new Proxy(super(...args), {
+			get(target, prop, receiver) {
+				// totally shit
```

Comment:
Is this expressing some sort of anger at the whole idea of using a `Proxy` here or just expressing that the current implementation needs work?

Reply:
The latter :) I just threw this together and knew there were going to be issues with this implementation.